### ISSUE TITLE
IllegalStateExceptionorg.mozilla.focus.search.RadioSearchEngineListPreference in onCheckedChanged

### ISSUE THREAD

**Issue by boek** [mozilla-mobile/focus-android#3602]
```
IllegalStateException

group.getChildAt(checkedId) must not be null

    org.mozilla.focus.search.RadioSearchEngineListPreference in onCheckedChanged at line 40
    Called from: android.widget.RadioGroup in setCheckedId
    org.mozilla.focus.search.SearchEngineListPreference in refreshSearchEngineViews at line 91
    org.mozilla.focus.search.SearchEngineListPreference in onBindViewHolder at line 47
    org.mozilla.focus.search.RadioSearchEngineListPreference in onBindViewHolder at line 28
    Called from: android.support.v7.preference.PreferenceGroupAdapter in onBindViewHolder
```

**Comment by sblatz** [comment]
Likely related to something with [this fix](https://github.com/mozilla-mobile/focus-android/pull/3668).

**Comment by sblatz** [comment]
After some discussion with @boek and consulting the sentry dashboard, I think it's safe to close this for now. We haven't had this crash on nightly, and it was almost certainly fixed by #3625 which added a null check making this crash impossible in the future. If we encounter it again we can re-open this!
